applications/nrf_desktop/doc/swift_pair_app.rst

```diff
@@ -0,0 +1,51 @@
+.. _nrf_desktop_swift_pair_app:
+
+Swift Pair module
+#################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The Swift Pair module is used to enable or disable the Swift Pair Bluetooth advertising payload depending on the selected Bluetooth peer (used local identity).
+The module distinguishes between dongle peer and general Bluetooth peers.
+
+Module events
+*************
+
+.. include:: event_propagation.rst
+    :start-after: table_swift_pair_app_start
+    :end-before: table_swift_pair_app_end
+
+.. note::
+    |nrf_desktop_module_event_note|
+
+Configuration
+*************
+
+The Swift Pair module requires enabling the following Kconfig options:
+
+* :kconfig:option:`CONFIG_BT_ADV_PROV_SWIFT_PAIR` - The nRF Desktop's :ref:`nrf_desktop_ble_adv` uses :ref:`bt_le_adv_prov_readme` to generate advertising and scan response data.
+  The Swift Pair advertising data provider (:kconfig:option:`CONFIG_BT_ADV_PROV_SWIFT_PAIR`) is used to add the Swift Pair payload to the advertising data.
+  The Swift Pair module uses API of the Swift Pair advertising data provider to enable or disable the Swift Pair advertising payload depending on application local identity that is in use.
```

Comment:
```suggestion
  Depending on the used local identity of the application, the Swift Pair module uses API of the Swift Pair advertising data provider to enable or disable the Swift Pair advertising payload.
```
Reshuffled a bit, let me know if the meaning is as intended

Reply:
Can we call it `application local identity` ? It is phrase that we use to distinguish local identities defined in the application from the ones defined in Bluetooth.

Reply:
updated as suggested :)